test/boost/database_test.cc

```diff
@@ -20,6 +20,7 @@
  */
 
 
+#include <boost/test/tools/old/interface.hpp>
```

Comment:
What's this? we don't include it anywhere else.

Reply:
Thanks ms-code editor :-/
It automatically adds includes it thinks are missing.

Reply:
@avikivity fixed in v3